docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3"
+
+services:
+  normandy:
+    image: mozilla/normandy:latest
+    command: >
+      bash -c "
+        /app/bin/wait-for-it.sh db:5432;
+        python ./manage.py migrate;
+        python ./manage.py update_actions;
+        python ./manage.py initial_data;
+        gunicorn normandy.wsgi:application
+      "
+    links:
+      - db
+      - autograph
+    networks:
+      - public_nw
+      - private_nw
+    environment:
+      DJANGO_DEBUG: "true"
+      DATABASE_URL: "postgres://postgres:postgres@db/normandy"
+      DJANGO_ALLOWED_HOSTS: "*"
+      DJANGO_CONFIGURATION: "ProductionInsecure"
+      DJANGO_SILENCED_SYSTEM_CHECKS: "security.W001,security.W003,security.W004,security.W008,security.W009,security.W012,security.W016,normandy.recipes.E006,dockerflow.health.W001,normandy.recipes.I001,normandy.recipes.I003"
+      DJANGO_AUTOGRAPH_URL: "http://autograph:8001"
```

Comment:
I think maybe there needs to be a slash here?

I'm getting 
```  File "/opt/venv/lib/python3.9/site-packages/requests/models.py", line 314, in prepare
    self.prepare_url(url, params)
  File "/opt/venv/lib/python3.9/site-packages/requests/models.py", line 382, in prepare_url
    raise InvalidURL(*e.args)
requests.exceptions.InvalidURL: Failed to parse: http://autograph:8001sign/data```

Reply:
Ah right, good catch. What did you do to make that happen? Or what was the rest of the stack trace?

Reply:
Just did a plain old `docker-compose up`

Reply:
Weird. I was only able to get this to happen by forcing a signing job with manage.py. Oh well, thanks for testing!